esmf_regrid/experimental/unstructured_regrid.py

```diff
@@ -99,9 +99,14 @@ def _as_esmf_info(self):
         nodeCoord = self.node_coords.flatten()
         nodeOwner = np.zeros([num_node])  # regridding currently serial
         elemId = np.arange(1, num_elem + 1)
-        elemType = self.fnc.count(axis=1)
-        # Experiments seem to indicate that ESMF is using 0 indexing here
-        elemConn = self.fnc.compressed() - self.nsi
+        if np.ma.isMaskedArray(self.fnc):
+            elemType = self.fnc.count(axis=1)
+            # Experiments seem to indicate that ESMF is using 0 indexing here
+            elemConn = self.fnc.compressed() - self.nsi
+        else:
+            elemType = self.fnc.shape[1] * np.ones(self.fnc.shape[0])
```

Comment:
I think the code here can be streamlined :  The multiply can be replaced by direct construction with np.full.
```suggestion
            elemType = self.full(fnc.shape[:1], self.fnc.shape[1])
```